client/src/app/components/pipeline/pipeline.component.ts

```diff
@@ -54,16 +57,27 @@ export class PipelineComponent {
     refetchInterval: 2000,
   }));
 
+  groupsQuery = injectQuery(() => ({
+    ...getGroupsWithWorkflowsOptions({ path: { repositoryId: this.repositoryId() || 0 } }),
+    refetchInterval: 2000,
```

Comment:
issue: please increase the refetch interval to at least 30s, otherwise the error handler will go crazy